packages/airnode-admin/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:14.17-alpine3.14
+
+ENV name="airnode-admin"
+ENV appDir="/app" \
+    appBin="/usr/local/bin/${name}"
+
+LABEL application=${name} \
+      description="Airnode Admin CLI"
+
+WORKDIR ${appDir}
+
+COPY --from=api3/airnode-artifacts /dependencies/common ${appDir}/node_modules
+COPY --from=api3/airnode-artifacts /dependencies/airnode-admin ${appDir}/node_modules
+COPY --from=api3/airnode-artifacts /packages ${appDir}/node_modules/@api3/
+COPY --from=api3/airnode-artifacts /build/packages/airnode-admin/dist ${appDir}/
+COPY packages/airnode-admin/docker/entrypoint.sh /entrypoint.sh
+
+RUN apk add --update --no-cache su-exec
+# Make the binary available within PATH
+RUN ln -s ${appDir}/bin/admin.js ${appBin} && chmod +x ${appBin}
+# Create Airnode user
+RUN adduser -h ${appDir} -s /bin/false -S -D -H ${name}
```

Comment:
You have to switch to that user with the `USER` directive in order to run the CLI under that user.

Reply:
Actually, do we need that user at all? Does docker run with sudo by default?

Reply:
The Docker service is usually run by root and base images like `node` here usually let you have root access so you can do whatever. But for the specific application image it's recommended to run the application under some non-root user.